api/graphql/makeModels.js

```diff
@@ -236,6 +236,7 @@ export default async function makeModels (userId, isAdmin, apiClient) {
       relations: [
         {activeMembers: { querySet: true }},
         {childGroups: {querySet: true}},
+        {customViews: {querySet: true}}, // Currently there is no queryset on the the actual graphQL type
```

Comment:
@tibetsprague If this is a 1-to-1 relationship, it doesn't need a queryset. Should I actually put quertset false here?

Reply:
hmm, no I dont think you do

Reply:
but how is it 1:1, groups can have multiple customViews right?

Reply:
As far as I understood it, there was only going to be one custom view per group in the design. Is that wrong or is the idea we want to architect for an expansion of customViews in the future?